integration/single-node-slug-deploy/check.go

```diff
@@ -31,7 +32,7 @@ func main() {
 	// 5. Verify that hello is running (listen to syslog? verify Runit PIDs? Both?)
 
 	// list of services running on integration test host
-	services := []string{"consul", "p2-preparer", "hello"}
```

Comment:
so since you have some uses of `services[1]` below, does that mean you need to change them to `services[0]`?